src/Pages/Events/EventCard.js

```diff
@@ -28,10 +27,7 @@ function EventCard(props) {
             {getDateWithSlashes(eventDate.slice(0, 10))} {startTime} - {endTime}
           </Row>
           <Row className='event-title'>{title}</Row>
-          <Row className='event-location'>
-            <Ionicon icon='md-pin' fontSize='1.5rem' color='#414141bd' />
-            {eventLocation}
-          </Row>
+          <Row className='event-location'>{eventLocation}</Row>
```

Comment:
import the function you created to show the map pin, and render it here like below
```suggestion
          <Row className='event-location'>
            {mapPinSymbol()}
            {eventLocation}
          </Row>
```